src/formatter.coffee

```diff
@@ -0,0 +1,12 @@
+define [], () ->
+  class Formatter
+
+    constructor: (@string) ->
+
+    sentenceCase: () ->
+      new_string = @string.replace('_', ' ')
```

Comment:
`#replace` only works on the first occurrence. Should this replace multiple `_`? e.g. `@string.split('_').join(' ')`